### PR TITLE
COD-14/COD-44

### PR DESCRIPTION
**Pull request by eldarht**

### CODE REVIEW

File: frontend/js/GUI/QualityMetricsWindow.js [hunk 1]
```diff
@@ -15,6 +17,7 @@ let qualityMetrics = (function (previousWindowSize, previousWindowPos){
         ImGui.SetNextWindowSize(new ImGui.ImVec2(290, canvas.height/4), ImGui.Cond.Always);
         ImGui.SetNextWindowCollapsed(false);
         ImGui.Begin("Quality Metrics");
+        ImGui.Text("Number of classes:      " + classCount);
```
Comment (zohaib194) [inline frontend/js/GUI/QualityMetricsWindow.js]: should use locale

File: frontend/js/GUI/QualityMetricsWindow.js [hunk 2]
```diff
@@ -39,9 +42,14 @@ let qualityMetrics = (function (previousWindowSize, previousWindowPos){
         return this.wSize;
     }
 
+    function setClassCount(count){
+        classCount = count;
```
Comment (Xillez) [inline frontend/js/GUI/QualityMetricsWindow.js]: Input validation? >= 0

File: frontend/js/GUI/QualityMetricsWindow.js [hunk 3]
```diff
@@ -43,6 +43,15 @@ let qualityMetrics = (function (previousWindowSize, previousWindowPos){
...
     function setClassCount(count){
+        if (count === "undefined") {
+            console.log(LOCALE.getSentence("generic_undefined"));
+            return;
+        }else if (count < 0) {
+            console.log(LOCALE.getSentence("generic_negative"));
+            return;
+        }
+
+        if (true) {}
```
Comment (Xillez) [inline frontend/js/GUI/QualityMetricsWindow.js]: Why?

File: frontend/js/GUI/QualityMetricsWindow.js [hunk 4]
```diff
@@ -43,6 +43,15 @@ let qualityMetrics = (function (previousWindowSize, previousWindowPos){
     }
 
     function setClassCount(count){
+        if (count === "undefined") {
+            console.log(LOCALE.getSentence("generic_undefined"));
+            return;
+        }else if (count < 0) {
```
Comment (Xillez) [inline frontend/js/GUI/QualityMetricsWindow.js]: MISSING SPACE!!!! AAAAAAAAAAAAAAHHHHHHHHHHHHHHH!!!!!!!!!!!!! :-)

**Pull request merged**